### PR TITLE
bootstrapping swagger to .net sdk tools

### PR DESCRIPTION
**Pull request by shahabhijeet**
Set of files required to kick off swagger to sdk

### CODE REVIEW

**Review by sergey-shandar — COMMENTED** [review]
I know that this is just a draft but I would propose to download `nuget.exe` instead of adding it to source control.

**Comment by shahabhijeet** [comment]
Yes I agree with you, if we can download it, that will be the best option. Let me give that a quick try and let's see if we can eliminate one binary out of our PR.
The worst case due to time constraint, we would like to start integrating this and trust me I would like to remove my static dependency off binaries as much as possible.

File: stostools/Directory.Build.props
Comment (daschult) [inline stostools/Directory.Build.props]:
```suggestion
		<DefaultCurrentUserTargetsFileName>$(LocalDevUser).bootStrap.targets</DefaultCurrentUserTargetsFileName>
```

File: stostools/Directory.Build.props
Comment (daschult) [inline stostools/Directory.Build.props]:
```suggestion
		<LocalDevUserBootstrapPropsFile>$(LocalDevUserTargetsDir)\$(DefaultCurrentUserTargetsFileName)</LocalDevUserBootstrapPropsFile>
```

File: stostools/NuGet.Config
Comment (daschult) [inline stostools/NuGet.Config]: Are you sure you want a reference to \\scratch2\ here?

**Review by ghost — APPROVED** [review]

File: stostools/NuGet.Config
Comment (shahabhijeet) [reply stostools/NuGet.Config]: Yes that is the best option we have right now for a quick turnaround time.
But the plan is to have a nuget package in nuget.org signed a published.
But this early in this cycle is not an optimum solution when we are doing this for the first time.

But rest assured, the only way we want to roll out is in the form of a signed nuget package (versioned)

File: stostools/Directory.Build.props
Comment (shahabhijeet) [reply stostools/Directory.Build.props]: @daschult from the github UI, I am not able to understand what is the comment on the diff.
So can you explain what is that you would like to change.

Comment (daschult) [reply stostools/Directory.Build.props]: Oh, I expected `Currentuser` to have a capital-u (`CurrentUser`).

Comment (shahabhijeet) [reply stostools/Directory.Build.props]: @daschult you are right your suggestion makes sense to have that proper casing. Will fix it if required in my next iteration, as this file is shared with our common tools, I will need to fix throughout and not just this repository.
Will keep the casing as is for now, but thanks for pointing it out.

**Comment by shahabhijeet** [comment]
@salameer can you sign off on this PR.

**Comment by salameer** [comment]
@daschult are we waiting for something to merge this?

@sergey-shandar why is this check not completed yet? is this related to the similar issues we've been having on other PRs?

**Comment by daschult** [comment]
@salameer @shahabhijeet mentioned that he didn't want to merge this PR until you approved it. That's the only reason I haven't merged it already.

**Review by salameer — CHANGES_REQUESTED** [review]
@shahabhijeet Can you please add your new stostools under the "Scrpts folder" I would like to avoid having more folders at the top level of the repo

**Comment by sergey-shandar** [comment]
@shahabhijeet It looks like Travis-CI is stuck. I don't have a link to Travis-CI job so the easiest way would be to make a dummy commit.

**Comment by shahabhijeet** [comment]
@salameer will this work?
\scripts\stostools\< all files under this folder >

**Comment by shahabhijeet** [comment]
@salameer @sergey-shandar is travis CI waiting for anything else? I added the commit that should have done the trick?

**Comment by shahabhijeet** [comment]
Closing this one and opening a new one for travis to kick in #5334

**Pull request closed without merging**
